src/client/src/components/Details/index.tsx

```diff
@@ -92,7 +93,7 @@ const Details = ({
             role="button"
             tabIndex={0}
             onClick={handleBackClick}
-            onKeyDown={keyDownHandler}
+            onKeyUp={keyDownHandler}
```

Comment:
- Why did you change from `onKeyDown` to `onKeyUp`?
- If we have to change it to `onKeyUp`, we should probably update the function to be called `keyUpHandler` instead.

Reply:
oh my bad, good catch, this wasn't something I was modifying but I mistook it as my code.